hnn_core/gui/_viz_manager.py

```diff
@@ -116,17 +117,21 @@ def unlink_relink(attribute):
     def _unlink_relink(f):
         @wraps(f)
         def wrapper(self, *args, **kwargs):
-            # Unlink the widgets using the provided link object
-            link_attribute: link = getattr(self, attribute)
-            link_attribute.unlink()
+            try:
+                # Unlink the widgets using the provided link object
+                link_attribute: link = getattr(self, attribute)
+                link_attribute.unlink()
 
-            # Call the original function
-            result = f(self, *args, **kwargs)
+                # Call the original function
+                result = f(self, *args, **kwargs)
 
-            # Re-link the widgets
-            link_attribute.link()
+                # Re-link the widgets
+                link_attribute.link()
 
-            return result
+                return result
+            except Exception as e:
```

Comment:
Usually you specify what kind of exception you are trying to catch and handle here. Is there a specific exception that you are encountering?

Reply:
Some GUI exceptions arent catch, like when out of bound or when it tries to use variables that haven't been set up yet. 
This was my way to recognize the exception's message. Should I ditch these lines or swap them for something a bit more polished?

Reply:
I see. If this is a good way for us to debug during development let's just keep this in for now.

Reply:
Maybe not all executions are logged to the console. Only script that is executed within a context manager of the logging window are logged. One of the first bugs I looked into was a plotting element that wasn't set within a context manager...